content/news/2020-XX-XX-keyfinder-gplv3.md

```diff
@@ -0,0 +1,32 @@
+title: New in 2.3: KeyFinder support and license change to GPL v3
+authors: Jan Holthuis
+tags: license, key detection, 2.3, beta
+status: draft
+
+Starting with the 2.3, Mixxx will be licensed under the terms of the [GNU General Public License (GNU GPL) version 3](https://www.gnu.org/licenses/rms-why-gplv3.html).
+This step is necessary because it allows us to integrate support for the excellent [KeyFinder library](https://github.com/mixxxdj/libKeyFinder) by Ibrahim Sha'ath.
+
+[KeyFinder](http://www.ibrahimshaath.co.uk/keyfinder/) is a pretty accurate open-source key detection tool.
+It's been in the top 3 of the [2020 key detection comparison on Reddit](https://www.reddit.com/r/DJs/comments/hwlzyt/key_detection_comparison_2020/) and performs considerably better than the key analyzer from the [QM-DSP library](https://code.soundsoftware.ac.uk/projects/qm-dsp) that current stable version of Mixxx is using.
+
+Unfortunately, it's also a lot slower than the QM-DSP algorithm (because the latter divides the sample rate by 8 and only analyzes every 8th window, resulting in a speedup factor of 64).
+We hope to improve performance in the future.
+Until then, the Queen Mary key Analyzer will be selected by default and you'll have to enable KeyFinder in the settings.
+
```

Comment:
Unfortunately we are unable to include KeyFinder in the Windows build of Mixxx 2.3.0 because we do not currently have a working server for building the Windows dependencies. If you want to help solve this, please get in touch on [Zulip](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Windows.20build.20environment.20on.20CI).

Reply:
Let me check if I can fix it using `vcpkg`, it looks pretty straighforward. If not, I'll add that paragraph.